### ISSUE TITLE
[6.0.0-rc.5] Wrongly nested additional props in Fields component

### ISSUE THREAD

**Issue by jdreesen** [erikras/redux-form#1583]
The docs say: `Any additional props that you pass to Field will be included at the root of the props structure given to your component`

```jsx
<Fields 
  names={[ 
    'field1',
    'email',
    // ...
  ]}
  component={MyMultiFieldComponent}
  anotherCustomProp="Some other information"/>

...the props given to your component would be of the structure:
{
  name: { input: { ... }, meta: { ... } },
  email: { input: { ... }, meta: { ... } },
  // ...
  anotherCustomProp: 'Some other information'
}
```

But instead the additional props are nested in each field like:

```jsx
{
  name: { input: { ... }, meta: { ... }, anotherCustomProp: 'Some other information' },
  email: { input: { ... }, meta: { ... }, anotherCustomProp: 'Some other information' },
  // ...
}
```

**Comment by erikras** [comment]
Great reporting, @jdreesen. 👍 Indeed the docs are the way I meant for it to be. Fixed in #1589. I'll comment here again when the fix is released.

**Comment by jdreesen** [comment]
Thx for the quick fix and for this awesome library! :)

**Comment by erikras** [comment]
Published in [`v6.0.1`](https://github.com/erikras/redux-form/releases/tag/v6.0.1).
